easypy/colors.py

```diff
@@ -334,7 +336,7 @@ def colorize(text):
 
     def _subfunc(match_obj):
         colorizer = Colorizer.from_markup(match_obj.group(1))
-        return colorizer(match_obj.group(2) or match_obj.group(3))
+        return colorizer(next(filter(None, match_obj.groups()[1:])))
```

Comment:
Is it possible to get a `StopIteration` exception here (less than 2 groups)?

Reply:
no, the regex has 4 groups (some may be none).
and the regex wouldn't match if at least one of them isn't populated.